This patch fixes the warnings:

mqueue.c: In function 'service_request':
mqueue.c:270: warning: incompatible implicit declaration of built-in function 'malloc'
mqueue.c: In function 'launch':
mqueue.c:305: warning: incompatible implicit declaration of built-in function 'free'
mqueue.c: In function 'scan_events':
mqueue.c:321: warning: incompatible implicit declaration of built-in function 'free'

---
 src/mqueue.c |    1 +
 1 file changed, 1 insertion(+)

Index: src/mqueue.c
===================================================================
--- src/mqueue.c.orig
+++ src/mqueue.c
@@ -26,6 +26,7 @@
 #include <sys/syscall.h>
 #include <unistd.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <limits.h>
 #include <stdarg.h>
 #include <errno.h>
